imod/mf6/model_gwf.py

```diff
@@ -200,9 +201,8 @@ def from_imod5_data(
             object containing the conductivity distribution options per package type.
             If you want a package to have a different allocation option,
             then it should be imported separately
-        regridder_types: Optional[dict[str, dict[str, tuple[RegridderType, str]]]]
-            the first key is the package name. The second key is the array name, and the value is
-            the RegridderType tuple (method + function)
+        regridder_types:  defaultdict[str, Optional[RegridMethodType]]
+            the key is the package name. The value is a subclass of RegridMethodType.
```

Comment:
Docstring doesn't match update of defaultdict to regular dict yet

Reply:
done